source/cloud/cloud-administration/converting-oauth-2.0-to-openid-connect.rst

```diff
@@ -0,0 +1,26 @@
+
+Converting OAuth 2.0 to OpenID Connect
+=======================================
+
+OAuth 2.0 is being deprecated in the System Console and replaced by OpenID Connect. Using the System Console, you can migrate your OAuth configuration to the OpenID Connect standard. 
```

Comment:
```suggestion
OAuth 2.0 is being deprecated and replaced by OpenID Connect. Using the System Console, you can migrate your OAuth configuration to the OpenID Connect standard. 
```

Reply:
@justinegeffen In this specific case, OAuth 2.0 is being deprecated only in the UI (the System Console). System Admins aren't being forced to migrate their existing OAuth 2.0 configurations to OpenID Connect (though we give them a quick way to do so). Given this context, would you continue to recommend that we remove the reference to the System Console here?

Reply:
Thanks for the context, and no, agreed that this would not work.